docs/core/versions/selection.md

```diff
@@ -0,0 +1,110 @@
+---
+title: .NET Core version selection
+description: Learn How .NET Core finds and chooses runtime versions for your program.
+author: billwagner
+ms.author: wiwagn
+ms.date: 06/27/2018
+---
+# .NET Core version selection
+
+[!INCLUDE [topic-appliesto-net-core-2plus](../../../includes/topic-appliesto-net-core-2plus.md)]
+
+This article explains the policies used by the .NET Core tools, SDK, and runtime for selecting versions. These policies provide a balance between running applications using the specified versions and enabling ease of upgrading both developer and end user machines. These policies perform the following:
+
+- Easy and efficient deployment of .NET Core, including security and reliability updates.
+- Usage of the latest tools and commands independent of target runtime.
+
+There are four activities during development and deployment where version selection occurs:
+
+- When you [run an SDK command](#the-sdk-uses-the-latest-installed-version).
+- When you [build an assembly](target-framework-monikers-define-build-time-apis).
+- When you [run a .NET Core application](framework-dependent-apps-roll-forward).
+- When you publish a [self-contained application](self-contained-deployments-include-the-selected-runtime).
```

Comment:
This is confusing, because the descriptions don't correspond to the headers, and these categories aren't mutually exclusive. For example, running an SDK command (item 1) includes building an assembly (item 2).